net/openvpn/patches/102-Disable-external-ec-key-support-when-building-with-wolfSSL.patch

```diff
@@ -0,0 +1,20 @@
+--- a/src/openvpn/ssl_openssl.c
++++ b/src/openvpn/ssl_openssl.c
```

Comment:
Can we import the patch with all the Git details, so we really know that it is backported patch and as well, we will know what it does?

You should import like this:
https://github.com/OpenVPN/openvpn/commit/4cf01c8e4381403998341aa32f79f4bf24c7ccb1.patch

Reply:
It's no backport. This fix initially made by me and later included in other MRs

Reply:
This patch was not backported. I only created it following the patch for LibreSSL you mentioned.

But of course I can try to get these patches upstream and put the patches incl. commit message etc. here.

Reply:
Ooops! My bad. These patches looks almost the same, but they are for different SSL library. indeed. Sorry guys.

But yeah, please upstream them if possible. I do not use WolfSSL either, so I can not help you with that one. I don't like when we include patches here without any commit message, etc. here, because we would end up with the same situation as it is in the main repo.